front/public/locales/de/operationalStudies/drivertrainschedule.json

```
@@ -0,0 +1,27 @@
+{
+  "actualspeed": "Tatsächlich",
+  "actualspeed-short": "Tat.",
+  "averagespeed": "Durchschnitt",
+  "averagespeed-short": "Dur.",
+  "composition": "Zusammensetzung",
+  "dcmdetails": "Zugverlauf n°",
+  "destination": "Ziel",
+  "energyconsumed-basic": "Kürzeste Laufzeit",
+  "energyconsumed-eco": "Energieschonendste Laufzeit",
+  "energydetails": "Energieumsatz am Radumfang",
+  "mass": "Masse",
+  "numberoflines": "Linienanzahl",
+  "origin": "Ursprung",
+  "pk": "PK",
+  "place": "Anlage",
+  "placeholderline": "Nr.",
+  "rollingstock": "Schienenfahrzeug",
+  "speed": "Geschwindigkeit",
+  "speedlimit": "Höchstgeschwindigkeit",
+  "speedlimit-short": "vMax",
+  "speedLimitByTag": "Zusammensetzungsschlüssel",
+  "time": "Zeit",
+  "timetable": "Fahrplan",
+  "trackname": "Gleis",
```

```suggestion
  "trackname": "Strecke",
```

I'm not very confident on the "Strecke" word